spec/index.html

```diff
@@ -1055,7 +1057,8 @@ <h3>Algorithm</h3>
       <p>The following algorithm will run with a minimal number of iterations in each step
         for typical <a>input datasets</a>.
         In some extreme cases, the algorithm can behave poorly, particularly in <a href="#ca.5">Step 5</a>.
-        Implementations MUST prevent against potential denial-of-service attacks.
+        Implementations MUST prevent against potential denial-of-service attacks
```

Comment:
This predates this PR, but something's wrong with "prevent against".
```suggestion
        Implementations MUST defend against potential denial-of-service attacks
```